### PR TITLE
Issue/#4/AddIndexSupport

### PR DESCRIPTION
**Pull request by Amila17**
Support for Adding Indexes (Unique and Compound)
Updated TravelAgent class to HandleIndexes for the build script.

### CODE REVIEW

**Comment by feanz** [comment]
This should also deal with this I guess https://github.com/feanz/lhm.net/issues/1

File: src/lhm.net/Migrator.cs [hunk 1]
```diff
@@ -50,6 +50,28 @@ public void RenameColumn(string oldColumnName, string newColumnName)
...
+        {
+            InternalAddIndex(indexName, isUnique, indexDef.ToString());
+        }
+
+        public void AddCompoundIndex(string indexName, bool isUnique, params IndexDef[] indexInfo)
+        {
+            InternalAddIndex(indexName, isUnique, String.Join(", ", indexInfo.Select(ii => ii.ToString())));
+        }
+
+        private void InternalAddIndex(string indexName, bool isUnique, string columnDefinition)
```
Comment (feanz) [inline src/lhm.net/Migrator.cs]: Can this not just be an overload of AddIndex?

File: src/lhm.net/Migrator.cs [hunk 2]
```diff
@@ -50,6 +50,28 @@ public void RenameColumn(string oldColumnName, string newColumnName)
             _renameMaps.Add(new RenameMap(oldColumnName, newColumnName));
         }
 
+        public void AddIndex(string indexName, bool isUnique, IndexDef indexDef)
+        {
+            InternalAddIndex(indexName, isUnique, indexDef.ToString());
+        }
+
+        public void AddCompoundIndex(string indexName, bool isUnique, params IndexDef[] indexInfo)
```
Comment (feanz) [inline src/lhm.net/Migrator.cs]: Can this not just be an overload of AddIndex

File: src/lhm.net/Migrator.cs (hunk 1 above)
Comment (Amila17) [inline src/lhm.net/Migrator.cs]: Nice a private overload. Forgot about that. :)

File: src/lhm.net/Migrator.cs (hunk 2 above)
Comment (Amila17) [inline src/lhm.net/Migrator.cs]: Done

**Comment by feanz** [comment]
:shipit:

**Pull request merged**